ccp/lib/processors/pipeline_information/builds.py

```diff
@@ -67,6 +68,56 @@ def __init__(
                 token_from_mount=token_from_mount
             )
 
+    def get_unlisted_info_from_core_jenkins(
+            self, ordered_job_list, build_number, keys
+    ):
+        """
+        Gets unlisted information from core jenkins
```

Comment:
add example of keys, which can be requested, this is needed, as one doesn't have to go through the sample json response for using this method, since now you should have all the setup ready, its easier to read those keys and write it down here.

Reply:
Added examples

Reply:
Added